content/docs/reference/design-docs/timetable/index.en.md

```diff
@@ -119,7 +119,7 @@ rolling_stock_name: R2D2
 labels: ["INOUI", "TCHOU"]
 
 # used to select speed limits for simulation
-speed_limit_tags: ["MA100"]
+speed_limit_tags: "MA100"
```

Comment:
are there supposed to be multiple tags, or just one? STDCM takes multiple ones, and the standalone train schedule has only one. Also, the name of the field has to be consistent with its arity